indico/modules/events/contributions/templates/management/_contribution_report.html

```diff
@@ -72,7 +79,9 @@
                                 TODO
                             </td>
                             <td class="i-table">
```

Comment:
data attribute for sorting; `n/a` should be an empty string when it comes to sorting